great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -214,9 +215,11 @@ def get_batch_data_and_markers(
             batch_data = batch_spec.batch_data
             if isinstance(batch_data, str):
                 raise ge_exceptions.ExecutionEngineError(
-                    f"""PandasExecutionEngine has been passed a string type batch_data, "{batch_data}", which is illegal.
-Please check your config."""
+                    f"""PandasExecutionEngine has been passed a string type batch_data, "{batch_data}", which is \
```

Comment:
Stray `\` since this is in `"""`? Below also?

Reply:
@billdirks I added it on purpose (to avoid `PyCharm` line length warnings).  Thanks.

Reply:
I'm surprised pycharm is throwing a line length warning if the line is < our line length limit. 
Most python code I've looked at doesn't use `\` but `"` each line since python will concatenate them together.

Reply:
Fixed.